content/en/docs/tasks/manage-daemon/rollback-daemon-set.md

```diff
@@ -4,28 +4,29 @@ reviewers:
 title: Perform a Rollback on a DaemonSet
 content_template: templates/task
 weight: 20
+min-kubernetes-server-version: 1.7
 ---
 
 {{% capture overview %}}
 
-This page shows how to perform a rollback on a DaemonSet. 
+This page shows how to perform a rollback on a {{< glossary_tooltip term_id="daemonset" >}}.
 
 {{% /capture %}}
 
 
 {{% capture prerequisites %}}
 
-* The DaemonSet rollout history and DaemonSet rollback features are only
-  supported in `kubectl` in Kubernetes version 1.7 or later.
-* Make sure you know how to [perform a rolling update on a
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+You should already know how to [perform a rolling update on a
```

Comment:
nit: `... You should already know ...`
Possibly reword to something like: 
This task assumes that you understand how to [perform ....]().